api/src/indicators.py

```diff
@@ -142,6 +140,71 @@ def patch_indicator(
     )
 
 
+def generate_keyword_query(term):
```

Comment:
I like ripping this out into it's own function to generate this big query, its more modular that way. Idk if it's worth it right now, but if we get another query this big we should move these generate_query type functions out into their own file, something like query_utils.py so that they don't clutter up the main indicators.py file.

Reply:
We'll address at a later time. Good suggestion though!